modules/finance/src/main/java/com/opengamma/strata/finance/rate/swaption/CashSettlement.java

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.finance.rate.swaption;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import org.joda.beans.BeanDefinition;
+import org.joda.beans.ImmutableBean;
+import org.joda.beans.JodaBeanUtils;
+import org.joda.beans.MetaProperty;
+import org.joda.beans.Property;
+import org.joda.beans.impl.direct.DirectFieldsBeanBuilder;
+import org.joda.beans.impl.direct.DirectMetaBean;
+import org.joda.beans.impl.direct.DirectMetaPropertyMap;
+
+/**
+ * Defines the settlement type and settlement method of swaptions.
+ * <p>
+ * The settlement type is {@link SettlementType#CASH}, i.e., Cash amount is paid by the short party to the long party 
+ * at the exercise date.
+
+ */
+@BeanDefinition
+public final class CashSettlement implements SwaptionSettlement, ImmutableBean, Serializable {
+
```

Comment:
Fine for the current PR, but will need to be more detailed when we implemented CashSettled swaption (par yield and zero coupon methods at least).